forage-android/src/main/java/com/joinforage/forage/android/ForageSDK.kt

```diff
@@ -40,19 +41,35 @@ object ForageSDK : ForageSDKApi {
             )
         )
 
-        return TokenizeCardService(
-            okHttpClient = OkHttpClientBuilder.provideOkHttpClient(
-                bearerToken,
-                merchantAccount,
-                idempotencyKey = UUID.randomUUID().toString()
-            ),
-            httpUrl = ForageConstants.provideHttpUrl(),
-            logger = logger
-        ).tokenizeCard(
-            cardNumber = currentEntry.getPanNumber(),
-            customerId = customerId,
-            reusable = reusable
-        )
+        return when {
+            shouldTokenize(currentEntry) -> TokenizeCardService(
+                okHttpClient = OkHttpClientBuilder.provideOkHttpClient(
+                    bearerToken,
+                    merchantAccount,
+                    idempotencyKey = UUID.randomUUID().toString()
+                ),
+                httpUrl = ForageConstants.provideHttpUrl(),
+                logger = logger
+            ).tokenizeCard(
+                cardNumber = currentEntry.getPanNumber(),
+                customerId = customerId,
+                reusable = reusable
+            )
+            else -> {
+                logger.e(
+                    "PAN entry was invalid",
+                    attributes = mapOf(
+                        "merchant_ref" to merchantAccount,
+                        "customer_id" to customerId
+                    )
+                )
+                ForageApiResponse.Failure(listOf(ForageError(400, "invalid_input_data", "Invalid PAN entry")))
+            }
+        }
+    }
+
+    private fun shouldTokenize(panEntry: PanEntry): Boolean {
```

Comment:
Remove this function and always let someone attempt to tokenize the EBT card.

Reply:
Addressed in #72 to make it easier to see changes!